configs/balrogworker

```diff
@@ -0,0 +1,25 @@
+{
+    "us-east-1": {
+        "type": "balrog-worker",
```

Comment:
Can you use the same naming schema everywhere (balrogworker vs barlog-worker). It'd be harder to grep otherwise.